NOTES.md

```diff
@@ -0,0 +1,235 @@
+# Librix server API Guide _18/05/2021_
+
+Running locally is set to port 5000.
+
+## Table of Contents
+
+- [Errors](#errors)
+- [Users](#users)
+- [Books](#books)
+- [Matches](#matches)
+
+## Errors
+
+If there is an error in any of the functions we return =>
+
+```
+{
+  "error": {
+    "message": "The error message"
+  }
+}
+```
+
+## Users
+
+All user calls start with /users
+
+### /
+
+**get** => response is an array with the users as objects, this will be deprecated in production of the app
```

Comment:
do you really want to mention the get Users ? - we dont really need it in there, do we?

Reply:
deleted just now